Update eslint packages

mythmon opened the pull request.

rehandalal commented on recipe-server/package.json:
```diff
@@ -70,14 +70,14 @@
-    "eslint": "3.19.0",
-    "eslint-config-airbnb": "10.0.1",
-    "eslint-import-resolver-webpack": "0.7.0",
+    "eslint": "^3.19.0",
```
shouldn't these be pinned?

mythmon commented:
The lints that were leftover were because I forgot to commit the .eslint file I added to the old control interface. I also fixed the test failures I introduced. I disabled a few rules in the old interface to avoid making too many changes.
